include/nana/gui/filebox.hpp

```diff
@@ -103,6 +103,11 @@ namespace nana
 		{
 			return show();
 		}
+
+		/// Set a new title for the dialog
+		/// @param	string	a text for title
+		/// @return the old title.
+		::std::string title( ::std::string new_title);
```

Comment:
The argument type should be const reference. I hope `std::string_view` will be used in future though.

Reply:
OK